x12-edi/2.5/modules/ROOT/pages/index.adoc

```diff
@@ -60,25 +60,26 @@ For example, using the `005010` version and the 850 transaction set, the schema
 
 If you're using one or more custom schemas, refer to xref:x12-edi-connector-config-topics.adoc[XML Schema and Trading Partner Configuration] for complete information on setting your schema locations.
 
-== About X12 Connector Message Structure
+== About X12 EDI Connector Message Structure
 
 This connector enables the reading or writing of X12 documents into (or from) the canonical EDI message structure.
 This structure is a hierarchy of Java Maps and Lists. You use either DataWeave or code to manipulate them.
 Each transaction has its structure as defined in the schemas.
 
 The message itself contains the following keys, some of which only apply to either the read operation or the write operation, as indicated:
 
+
 [%header%autowidth.spread]
 |===
 |Key name |Description
 |Delimiters |The delimiters used for the message. If used, this set is based on the last interchange processed by a read operation. If set for a write operation, this overrides the values in the module configuration. The characters in the string are interpreted based on position in the following order: data separator, sub-element separator, repetition separator, or `U` if none, segment terminator.
```

Comment:
Reread and reword to avoid confusion between "this set" (of what?) vs. "If set for a write operation" (set=configured).

If "this set" = characters or string of characters, then use one of those terms instead.

My best guess:
```suggestion
|Delimiters |An optional set of delimiters used in the message. If used in a read operation, the delimiters are based on the most recent interchange processed by a read operation. If used for a write operation, the delimiters override the values in the module configuration. The characters in the string of delimiters are interpreted based on their position in string, in the following order: data separator, subelement separator, repetition separator, and segment terminator, with a value of `U` indicating no value in that position.
```
This is just a guess, but you can extrapolate from it a clearer way of expressing this definition I think.

Reply:
Note also that this is a LOT of text for a definition, much more than used for its peers that follow. Perhaps some of this belongs elsewhere, in a narrative description of delimiters, if there is one?